doc/publications/cooling.rst

```diff
@@ -0,0 +1,14 @@
+Cooling
+=======
+
+Deep seawater cooling and desalination: Combining seawater air conditioning and desalination
+--------------------------------------------------------------------------------------------
+.. figure:: ../_static/publication_figures/10.1016-j.scs.2021.103257.jpg
+   :width: 250px
+   :align: right
+
+**Hunt, J.D., Weber, N. de A. B., Zakeri, B. et al. (2021), Sustainable Cities and Society**
```

Comment:
Since we already use [sphinxcontrib-bibtex](https://sphinxcontrib-bibtex.readthedocs.io/en/latest/usage.html), we should use its features and avoid manually formatting citations.